From 6634be73e5547d1e00320eaa3218f77b96aaefb5 Mon Sep 17 00:00:00 2001
From: Xie Xiaodong <xiexiaodong16@huawei.com>
Date: Sat, 11 Oct 2025 11:18:11 +0800
Subject: [PATCH 775/960] downstream: config: enable kernel config for vm
 memory hotplug

vm memory hotplug needs enable some kernel config:
  CONFIG_ACPI_HOTPLUG_MEMORY=y
  CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y

Signed-off-by: Xie Xiaodong <xiexiaodong16@huawei.com>
Signed-off-by: Liu Shixin <liushixin2@huawei.com>
---
 config.aarch64     | 4 ++--
 config.aarch64-64k | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/config.aarch64 b/config.aarch64
index 6e60599dd8b2..a882970db60b 100644
--- a/config.aarch64
+++ b/config.aarch64
@@ -657,7 +657,7 @@ CONFIG_ACPI_TABLE_UPGRADE=y
 # CONFIG_ACPI_DEBUG is not set
 CONFIG_ACPI_PCI_SLOT=y
 CONFIG_ACPI_CONTAINER=y
-# CONFIG_ACPI_HOTPLUG_MEMORY is not set
+CONFIG_ACPI_HOTPLUG_MEMORY=y
 CONFIG_ACPI_HED=y
 # CONFIG_ACPI_CUSTOM_METHOD is not set
 CONFIG_ACPI_BGRT=y
@@ -1059,7 +1059,7 @@ CONFIG_EXCLUSIVE_SYSTEM_RAM=y
 CONFIG_ARCH_ENABLE_MEMORY_HOTPLUG=y
 CONFIG_ARCH_ENABLE_MEMORY_HOTREMOVE=y
 CONFIG_MEMORY_HOTPLUG=y
-# CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE is not set
+CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y
 # CONFIG_MEMORY_HOTREMOVE is not set
 CONFIG_MHP_MEMMAP_ON_MEMORY=y
 CONFIG_ARCH_MHP_MEMMAP_ON_MEMORY_ENABLE=y
diff --git a/config.aarch64-64k b/config.aarch64-64k
index 1b8861f11c6d..4706984f2437 100644
--- a/config.aarch64-64k
+++ b/config.aarch64-64k
@@ -659,7 +659,7 @@ CONFIG_ACPI_TABLE_UPGRADE=y
 # CONFIG_ACPI_DEBUG is not set
 CONFIG_ACPI_PCI_SLOT=y
 CONFIG_ACPI_CONTAINER=y
-# CONFIG_ACPI_HOTPLUG_MEMORY is not set
+CONFIG_ACPI_HOTPLUG_MEMORY=y
 CONFIG_ACPI_HED=y
 # CONFIG_ACPI_CUSTOM_METHOD is not set
 CONFIG_ACPI_BGRT=y
@@ -1059,7 +1059,7 @@ CONFIG_EXCLUSIVE_SYSTEM_RAM=y
 CONFIG_ARCH_ENABLE_MEMORY_HOTPLUG=y
 CONFIG_ARCH_ENABLE_MEMORY_HOTREMOVE=y
 CONFIG_MEMORY_HOTPLUG=y
-# CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE is not set
+CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y
 # CONFIG_MEMORY_HOTREMOVE is not set
 CONFIG_MHP_MEMMAP_ON_MEMORY=y
 CONFIG_ARCH_MHP_MEMMAP_ON_MEMORY_ENABLE=y
-- 
2.43.0

